### DISCUSSION TITLE
rDNS internal for multi server not working

### DISCUSSION THREAD

**Discussion by KGBist2000** [AdguardTeam/AdGuardHome#3094] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x ] I am running the latest version
- [ x] I checked the documentation and found no answer
- [ x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.106.1
* **How did you install AdGuard Home:**
  * Github releases
* **How did you setup DNS configuration:**
  * LAN-Clients: 192.168.76.0/24
  * VPN-Clients: 10.0.0.0/24
  * FritzBox: Gateway + DHCP + DNS for fritz.box: 192.168.76.1
  * AdGuard Home: 192.168.76.3:53
  * Unbound-Resolver (local on ADH-Machine) + DNS for home.vpn: 127.0.0.1:5335
* **If it's a router or IoT, please write device model:**
  * Fritzbox 7490
* **CPU architecture:**
  * Intel x86
* **Operating system and version:**
  * Proxmox Host + Ubuntu 20.04 LXC for ADH + Unbound

### Expected Behavior
Definition of more the one local DNS-Server to resolve different domains for ADH-Webseite (Clientlist).

### Actual Behavior
The resolution in the local network is working well.
nslookup 10.0.0.10 -> iPhone.home.vpn
nslookup 192.168.76.10 -> iPhone.fritz.box

In the Client-List of ADH for IPV6 and 10.0.0.0/24 the Resolution is not working and only the IPs are showed.
after different tries, only the first defined local DNS-Server in the list is taken into account for PTR requests.

### Screenshots
<details><summary>Screenshot:</summary>

<img width="585" alt="Bildschirmfoto 2021-04-30 um 00 45 01" src="https://user-images.githubusercontent.com/81825812/116784684-e8f2b000-aa95-11eb-8c65-1fcdc084547d.png">
<img width="590" alt="Bildschirmfoto 2021-04-30 um 00 45 32" src="https://user-images.githubusercontent.com/81825812/116784688-ebeda080-aa95-11eb-902d-12ae8892eb3b.png">
<img width="590" alt="Bildschirmfoto 2021-04-30 um 00 46 29" src="https://user-images.githubusercontent.com/81825812/116784693-f314ae80-aa95-11eb-9fdb-6e0f13870c41.png">

</details>

**Comment by ameshkov** [comment]
If you want to use multiple servers in Private DNS, you're supposed to use this notation:
https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#specifying-upstreams-for-rdns
